examples/minimal-winit-android/src/lib.rs

```diff
@@ -0,0 +1,132 @@
+#[cfg(target_os = "android")]
+use winit::platform::android::activity::AndroidApp;
+
+use pixels::{Pixels, SurfaceTexture};
+use winit::event::Event;
+use winit::event_loop::{ControlFlow, EventLoop, EventLoopBuilder};
+use winit::window::Window;
+
+const WIDTH: u32 = 320;
+const HEIGHT: u32 = 240;
+const BOX_SIZE: i16 = 64;
+
+/// Representation of the application state. In this example, a box will bounce around the screen.
+struct World {
+    box_x: i16,
+    box_y: i16,
+    velocity_x: i16,
+    velocity_y: i16,
+}
+
+fn _main(event_loop: EventLoop<()>) {
+    let mut window: Option<Window> = None;
+    let mut pixels: Option<Pixels> = None;
+
+    let mut world = World::new();
+
+    event_loop.run(move |event, event_loop, control_flow| {
+        *control_flow = ControlFlow::Wait;
+        match event {
+            Event::Resumed => {
+                let _window = Window::new(&event_loop).unwrap();
+                let _pixels = {
+                    let window_size = _window.inner_size();
+                    let surface_texture =
+                        SurfaceTexture::new(window_size.width, window_size.height, &_window);
+                    Pixels::new(WIDTH, HEIGHT, surface_texture).unwrap()
+                };
+                _window.request_redraw();
+                window = Some(_window);
+                pixels = Some(_pixels);
+            }
+            Event::Suspended => {
+                window = None;
+                pixels = None;
+            }
+            Event::RedrawRequested(_) => {
+                if let Some(_pixels) = &mut pixels {
+                    if let Some(_window) = &window {
+                        world.draw(_pixels.frame_mut());
+                        _pixels.render().unwrap();
+                        _window.request_redraw();
+                    }
+                }
+            }
+            _ => {}
+        }
+        if let Some(_window) = &window {
```

Comment:
This should use `.is_some()` instead of pattern matching. I suspect you should see this one in the lint test results.

```suggestion
        if window.is_some() {
```